src/features/multivariate_layer/readme.md

```
@@ -0,0 +1,79 @@
+5. `stepOverrides` - only used for bivariate coloring. Defines value ranges for bivariate colors
+6. `opacity` - Opacity dimension. Contains MCDA layer config. Final score is used to apply different levels of opacity to different features.
+7. `text` - Text (Label) dimension. Can either display text based on MCDA config or using MapLibre expression.
+
+- `mcdaValue` - Contains MCDA layer config. The output depends on `mcdaMode`
+- `mcdaMode` property:
+  - `score` - Single line of text (total MCDA score)
+  - `layers` - One line of text for each of the MCDA axes (layers) in MCDA config
+- `expressionValue` - optional property, can contain MapLibre expression, for custom outputs and calculations. `mcdaValue` overwrites this.
+- `formatString` - optional string template. Replaces `{value}` and `{unit}` with MCDA axis value and unit accordingly. If not present, `{value} {unit}` template is used.
+- `precision` - optional, number. How many numbers is retained after the decimal point.
+- `paintOverrides` and `layoutOverrides` - optional, can be used to customize MapLibre paint and layer specification. See for more at https://maplibre.org/maplibre-style-spec/layers/#symbol
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)